avm/utl/types/avm-common-types/tests/e2e/import/main.test.bicep

```diff
@@ -268,11 +268,11 @@ param secretToSet secretToSetType[] = [
 #disable-next-line outputs-should-not-contain-secrets // Does not contain a secret
 output secretToSetOutput secretToSetType[] = secretToSet
 
-param secretSet secretSetType[] = [
+param secretSet secretSetOutputType[] = [
```

Comment:
changing a param name qualifies for a minor bump right?

Reply:
I did not expect anybody to use this feature yet - but you're right. Updating

Reply:
done